webapp/src/widgets/menu/menu.tsx

```diff
@@ -31,9 +32,9 @@ export default class Menu extends React.PureComponent<Props> {
     }
 
     public render(): JSX.Element {
-        const {position, children} = this.props
+        const {position, fixed, children} = this.props
         return (
-            <div className={'Menu noselect ' + (position || 'bottom')}>
+            <div className={`Menu noselect ' ${position || 'bottom'} ${fixed && ' fixed'}`}>
```

Comment:
Appears to be some mixed quotation marks....see the unit tests.

Reply:
@sbishel Should be fixed.